.evergreen/config.yml

```diff
@@ -78,11 +83,13 @@ functions:
           CLUSTER_NAME_SALT: ${build_id}
           ATLAS_API_USERNAME: ${atlas_key}
           ATLAS_API_PASSWORD: ${atlas_secret}
-          ATLAS_API_BASE_URL: ${atlas_url}
```

Comment:
With the private endpoints the last path segment had to be removed, since I didn't have admin access I set the override in the PR. I can edit the project but this would break existing builds, therefore I suggest making this change in project config when the PR is ready to be merged.

Reply:
Ok. In that case can you revert this change? We can update the `atlas_url` evergreen variable before merging this.